BeatSaberHTTPStatus/manifest.json

```diff
@@ -0,0 +1,13 @@
+﻿{
+  "$schema": "https://raw.githubusercontent.com/beat-saber-modding-group/BSIPA-MetadataFileSchema/master/Schema.json",
+  "author": "opl-",
```

Comment:
It's "opl", without the dash. As it turns out getting 3 character usernames is unfortunately nearly impossible on most websites nowadays. :/